### PR TITLE
Login register connect

### PR DESCRIPTION
**Pull request by HiImZsofi**
fogadd mar el he

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@import "bootstrap/dist/css/bootstrap.css";
```
Comment (BAkosM) [inline client/src/App.css]: Delete this

File: client/src/App.tsx [hunk 2]
```diff
@@ -1,29 +1,32 @@
 //imports
-import React, { useEffect, useState } from 'react';
-import logo from './logo.svg';
-import './App.css';
-import { response } from 'express';
```
Comment (lacgondi) [inline client/src/App.tsx:5]: Remove unnecessary imports

File: client/src/Views/UserSettings.tsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import { Navbar } from "react-bootstrap";
+import FormSubmitButton from "../components/FormSubmitButton";
+import FormSwitch from "../components/FormSwitch";
+import FormWrapper from "../components/FormWrapper";
+import InputField from "../components/InputField";
+import NavMenu from "../components/NavBar";
+import { UserSettingsDto } from "../Interfaces";
+
+class UserSettings extends React.Component<{}, UserSettingsDto> {
```
Comment (lacgondi) [inline client/src/Views/UserSettings.tsx:9]: Add todo "Fix the input fields"

File: client/src/components/navBar.tsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  class: string;
+} 
+let NavMenuPoints: MenuPoints[] = [
+  { text: "Pontjaim", link: "./userPage", class: "me-2"},
+  { text: "Rangsor", link: "./rankPage", class: "me-2"},
+  { text: "Barátok", link: "./friendPage", class: "me-auto"},];
+
+//Navbar
+const NavMenu = ()=> {
+  const [id, setId] = React.useState(0);
```
Comment (lacgondi) [inline client/src/components/navBar.tsx]: Lift the states to the user pages

File: client/src/components/navBar.tsx [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+  const [username, setUserName] = React.useState("");
+  const [picfilepath, setPicfilepath] = React.useState("");
+  React.useEffect(() => {
+    const fetchData = async () => {
+    try {
+      const token = localStorage.getItem("token");
+      //TODO add fetch source 
+      const res = await fetch("", {
```
Comment (lacgondi) [inline client/src/components/navBar.tsx]: Fetch from the userpages instead

File: client/src/components/navBar.tsx [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+//Navbar
+const NavMenu = ()=> {
+  const [id, setId] = React.useState(0);
+  const [username, setUserName] = React.useState("");
+  const [picfilepath, setPicfilepath] = React.useState("");
+  React.useEffect(() => {
+    const fetchData = async () => {
+    try {
+      const token = localStorage.getItem("token");
+      //TODO add fetch source
```
Comment (BAkosM) [inline client/src/components/navBar.tsx]: Fix this

**Review by BAkosM — COMMENTED** [review]
Rename everything to uppercase letter

**Review by lacgondi — CHANGES_REQUESTED** [review]
rename files

**Review by HiImZsofi — COMMENTED** [review]
xd

**Review by lacgondi — APPROVED** [review]
seems ok

**Pull request merged**